cmd/contour/serve.go

```diff
@@ -199,6 +199,11 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 
 	listenerConfig.DefaultHTTPVersions = defaultHTTPVersions
 
+	responseTimeoutRange, err := timeout.ParseRange(ctx.ResponseTimeoutMin, ctx.ResponseTimeoutMax)
+	if err != nil {
+		return fmt.Errorf("failed to parse request timeout min/max: %w", err)
```

Comment:
Should we use the logger here to print out the error message? I know in other parts of serve.go we do the same so at least this is consistent.

Reply:
I *think* we want this one to be a fatal error, which is why I'm returning it rather than just logging it. I was following the pattern from just above for parsing the default HTTP version from the config file. Alternately, we could warn and resort to no limits.